Mathlib/Geometry/Manifold/Algebra/LeftInvariantDerivation.lean

```diff
@@ -163,12 +163,12 @@ theorem coe_zero : ⇑(0 : LeftInvariantDerivation I G) = 0 :=
   rfl
 #align left_invariant_derivation.coe_zero LeftInvariantDerivation.coe_zero
 
-@[simp]
+@[simp, nolint simpNF] -- timeouts when `import Mathlib`.
 theorem coe_neg : ⇑(-X) = -X :=
   rfl
 #align left_invariant_derivation.coe_neg LeftInvariantDerivation.coe_neg
```

Comment:
```
import Mathlib

universe u_4 u_3 u_2 u_1

set_option synthInstance.maxHeartbeats 20000 in
example {𝕜 : Type u_1} [NontriviallyNormedField 𝕜] {E : Type u_2}
  [NormedAddCommGroup E] [NormedSpace 𝕜 E] {H : Type u_3} [TopologicalSpace H]
  {I : ModelWithCorners 𝕜 E H} {G : Type u_4} [TopologicalSpace G] [ChartedSpace H G]
  [Monoid G] [SmoothMul I G] (X : LeftInvariantDerivation I G) : ⇑(-X) = -⇑X := by simp
```
already fails in current master

Reply:
Is this still the case? Maybe some miracles happened...

Reply:
Seems fixed.